diesel_derives/src/insertable.rs

```diff
@@ -87,7 +87,7 @@ pub fn derive(item: DeriveInput) -> TokenStream {
         {
             type Values = <(#(#direct_field_ty,)*) as Insertable<#table_name::table>>::Values;
 
-            fn values(self) -> Self::Values {
+            fn values(self) -> <(#(#direct_field_ty,)*) as Insertable<#table_name::table>>::Values {
```

Comment:
This doesn't need to change.

Reply:
It does need to change as otherwise rustc emits for whatever reason 2 error message, where one is pointing to the derive itself, and the other one to the corresponding field.